jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -131,15 +131,8 @@
 	static final Unsafe getUnsafe() {
 		return UnsafeGetter.myUnsafe;
 	}
```

Comment:
If we're going to change the field for JITHelpers, we might as well do the same for Unsafe. It uses the same approach.

Reply:
I did the change in the new commit, have to modify all method handle classes where `getUnsafe()` is called.